### PR TITLE
Feat/change request overview applied state

### PR DESCRIPTION
**Pull request by FredrikOseberg**
## About the changes

This PR adds 
* a dropdown button for the change request overview page that you can use to approve / reject changes
* minor styling changes

Skjermbilde 2022-11-03 kl  11 09 08

Closes #

### CODE REVIEW

File: frontend/src/component/changeRequest/ChangeRequestOverview/ReviewButton/ReviewButton.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                                    : 'center bottom',
+                        }}
+                    >
+                        <Paper>
+                            <ClickAwayListener onClickAway={onClose}>
+                                <MenuList
+                                    id="review-options-menu"
+                                    autoFocusItem
+                                >
+                                    {options.map((option, index) => (
```
Comment (kwasniew) [inline frontend/src/component/changeRequest/ChangeRequestOverview/ReviewButton/ReviewButton.tsx:117]: I'd simplify it by passing two explicit MenuItem components each with its own onMenuItemClick. Currently to track the logic I had to iterate over two strings and then visually filter out a correct path inside onMenuItemClick.

File: frontend/src/component/changeRequest/ChangeRequestOverview/ReviewButton/ReviewButton.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    const id = useRequiredPathParam('id');
+    const { refetchChangeRequest } = useChangeRequest(projectId, id);
+    const { setToastApiError, setToastData } = useToast();
+
+    const { changeState } = useChangeRequestApi();
+
+    const [open, setOpen] = React.useState(false);
+    const anchorRef = React.useRef<HTMLButtonElement>(null);
+
+    const onMenuItemClick = async (index: number) => {
```
Comment (kwasniew) [inline frontend/src/component/changeRequest/ChangeRequestOverview/ReviewButton/ReviewButton.tsx:33]: I'd split this into separate fns

**Review by kwasniew — APPROVED** [review]
looks good, one important comment left

File: frontend/src/component/changeRequest/ChangeRequestOverview/ReviewButton/ReviewButton.tsx (hunk 1 above)
Comment (FredrikOseberg) [reply frontend/src/component/changeRequest/ChangeRequestOverview/ReviewButton/ReviewButton.tsx:117]: Thanks! That is much better indeed. I got locked into material-uis example here.

File: frontend/src/component/changeRequest/ChangeRequestOverview/ReviewButton/ReviewButton.tsx (hunk 2 above)
Comment (FredrikOseberg) [reply frontend/src/component/changeRequest/ChangeRequestOverview/ReviewButton/ReviewButton.tsx]: Addressed here: https://github.com/Unleash/unleash/pull/2322/commits/2f766a26df55d64bc600020aa3e5f9020f10abdb

**Pull request merged**